### ISSUE TITLE
InfluxDB error: max-values-per-tag limit exceeded

### ISSUE THREAD

**Issue by tiltec** [bitlabstudio/django-influxdb-metrics#27]
After long-term use, this error appears on almost every request:

```
400: {"error":"partial write: max-values-per-tag limit exceeded (100000/100000): measurement=\"django_request\" tag=\"full_path\" value=\"/api/pickup-dates/20369/\" dropped=1"}
```

The `full_path` tag can vary much if there's IDs in the URL. While it would be possible to increase the limit [via configuration parameter](https://docs.influxdata.com/influxdb/v1.6/administration/config/#max-values-per-tag-100000), it might be better to change something about the storage pattern. It appears that having many different tag values [is not recommended](https://docs.influxdata.com/influxdb/v1.6/concepts/schema_and_data_layout/#don-t-have-too-many-series).

I made a quick fix by removing the `full_path` and `path` tags from written data.
A better way would be to write the URL _pattern_ as tag and the actual path as field.

**Comment by mbrochh** [comment]
Whoa, I haven't been using this library in quite a while. Would you fix be backwards compatible? Do you want to submit a pull request?

**Comment by tiltec** [comment]
Yeah, there's not much activity here. I wonder if people still use it? Most medium-sized deployments should hit the max-values-per-tag limit after some months.

As said, I simpliy removed the full_path and path tags in my fork. I could contribute that as pull request, but maybe somebody still wants to have the data?
